src/stories/lib/markdown-to-csf.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable import/no-extraneous-dependencies */
+// we disable this rule because this file is only meant to be processed by Storybook. It is not part of our npm / CDN bundle.
+import yaml from 'js-yaml';
+import frontmatter from 'remark-frontmatter';
+import remarkGfm from 'remark-gfm';
+import highlight from 'remark-highlight.js';
+import remark2Html from 'remark-html';
+import remarkParse from 'remark-parse';
+import unified from 'unified';
+
+export function rollupMdToCsfPlugin () {
+  return {
+    name: 'markdown-to-csf',
+    transform (code, id) {
+      if (id.endsWith('.md')) {
+
+        return {
+          code: markdownToCsfWithDocsPage(code),
+        };
+      }
+      return code;
+    },
+  };
+}
+
+const processor = unified()
+  .use(remarkParse, {})
+  .use(remarkGfm)
+  .use(frontmatter, ['yaml'])
+  .use(highlight)
+  .use(remark2Html, { sanitize: false });
+
+export function markdownToCsfWithDocsPage (markdownText) {
+
+  const markdownAst = processor.parse(markdownText);
+
+  const frontmatterNode = markdownAst.children.find((node) => node.type === 'yaml');
+  const kind = getKind(frontmatterNode);
+
+  const headingNode = markdownAst.children.find((node) => node.type === 'heading' && node.depth === 1);
+  const subtitle = getSubTitle(frontmatterNode, headingNode);
+
+  const title = [kind, subtitle].filter((a) => a != null).join('/');
+
+  const htmlContent = processor().processSync(markdownText).contents;
+  const parsedHTML = JSON.stringify(htmlContent);
+
+  const csfScript = `
+    import { MarkdownDocs } from '/src/stories/lib/markdown-docs.jsx';
+    import React from 'react';
+
+    export default {
+      title: '${title}',
+      parameters: {
+        docs: {
+          page: () => React.createElement(MarkdownDocs, { html: ${parsedHTML} }),
```

Comment:
Using an external `jsx` component is not mandatory but it makes the file easier to read so that's the option I prefer.

If you want to revert this back to what we did before (see https://github.com/CleverCloud/clever-components/blob/master/src/stories/lib/markdown.cjs#L72), this should be doable so don't hesitate to say it here :wink: